### PR TITLE
Toms controllers + simons exceotions java docs

### PR DESCRIPTION
**Pull request by jerdbaer**

### CODE REVIEW

File: controller/MainViewInputRestrictionController.java [hunk 1]
```diff
@@ -6,6 +6,13 @@
 import javafx.scene.control.TextFormatter;
 import javafx.scene.control.TextFormatter.Change;
 
+/*
+ * Program to validate the userer's input while user is typing in working and break times in user interface and displays
+ * all input (key or mouse driven) in the related 
+ * 
+ * @author Tom Weißflog
+ * @version 1.0
+ */
```
Comment (TomxHWRxWI20C) [inline controller/MainViewInputRestrictionController.java:15]: This provides rules to restrict the possible Keyinput (includes Strg+V or Mouse copy)  in Textfields
- only Numbers
- only hours (0-23)
- only minutes(0-59)

File: controller/OverviewController.java [hunk 2]
```diff
@@ -60,7 +74,12 @@ else if(pressedBtn == btnLogout) {
     	}
 
     }
-    
+ //------------------must be updated -------------------   
+    /**
+     * Displays working time data for declared user of declared time period (current week, current month or earlier month)
+     * @param event button click "on"
+     */
```
Comment (TomxHWRxWI20C) [inline controller/OverviewController.java:81]: change earlier month to random data

File: controller/PopupDoubleEntryController.java [hunk 3]
```diff
@@ -81,6 +97,11 @@ void abortToWorktime(ActionEvent event) {
     	popup.hide();
 	}
 
+	/**
+     * Saves the created working day record for declared user ID to database and 
+     * closes popup window afterwards.
+     * @param event button click on "Speichern"
+     */
```
Comment (TomxHWRxWI20C) [inline controller/PopupDoubleEntryController.java:107]: this overwrites the old entry too

File: controller/PopupLimitsController.java [hunk 4]
```diff
@@ -9,7 +9,15 @@
 import javafx.scene.control.Label;
 import javafx.scene.control.TextField;
 
-
+/**
+ * Program to open a new popup window for user interaction.
+ * Warns about imminent double entry for working day record which results in an
+ * overwrite of existing record. Requires user decision how to proceed.
+ *
```
Comment (TomxHWRxWI20C) [inline controller/PopupLimitsController.java:16]: this is wrong 
this part is for workinglimits 
popup to add comment why worked so early / late

File: controller/SwapSceneController.java [hunk 5]
```diff
@@ -43,6 +56,10 @@ public void goToLogin() {
     	}
```
Comment (TomxHWRxWI20C) [inline controller/SwapSceneController.java]: needs to be deleted 
we don't want the methode goToLogin()

**Review by TomxHWRxWI20C — CHANGES_REQUESTED** [review]
only small change looks very good
nice one

**Review by TomxHWRxWI20C — APPROVED** [review]

**Review by valientes — APPROVED** [review]

**Pull request merged**